stats/src/main/java/io/airlift/stats/CpuTimer.java

```diff
@@ -66,24 +81,28 @@ public CpuDuration elapsedIntervalTime()
     {
         long currentWallTime = System.nanoTime();
         long currentCpuTime = THREAD_MX_BEAN.getCurrentThreadCpuTime();
-        long currentUserTime = THREAD_MX_BEAN.getCurrentThreadUserTime();
+        // ThreadMXBean will return -1 if user CPU time collection is not supported
+        long currentUserTime = (collectUserTime && intervalUserStart != -1) ? THREAD_MX_BEAN.getCurrentThreadUserTime() : -1;
```

Comment:
No need to check `intervalUserStart != -1` ?

Reply:
See explanation in https://github.com/airlift/airlift/pull/1024#discussion_r1051025454